### PR TITLE
Parser

### PR DESCRIPTION
**Pull request by pedrocunial**
My implementation for a input file parser with our program :octocat:

### CODE REVIEW

File: exercicio.py [hunk 1]
```diff
@@ -152,21 +139,9 @@ def calc_real_deal():
...
-P_g = [
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    -1000
-]
```
Comment (gabicfa) [inline exercicio.py:179]: Why did you delete this line???

File: exercicio.py [hunk 2]
```diff
@@ -184,8 +159,8 @@ def cut(matrix):
 
 matrix = cut(real_deal)
 
+
```
Comment (gabicfa) [inline exercicio.py]: Nice pep8

File: exercicio.py [hunk 3]
```diff
@@ -184,8 +159,8 @@ def cut(matrix):
...
 def calc_u():
-    # t_matrix = np.linalg.inv(matrix)  # invert
```
Comment (gabicfa) [inline exercicio.py]: Nice deletation cometation code

**Review by gabicfa — APPROVED** [review]

**Comment by gabicfa** [comment]
Tested and it worked!!!!!!!!!!!!!!!

:octocat:

**Review by marcelogdeandrade — APPROVED** [review]
